source/TestFrameworkShared/Helper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
+
+namespace nanoFramework.TestFramework
+{
+    public static class Helper
+    {
+        private static string GetJoinedParams(object[] data)
+        {
+            var returnString = string.Empty;
+            foreach (var item in data)
+            {
+                returnString += $"{item} | ";
+            }
+
+            return returnString.Substring(0, returnString.Length - 4);
+        }
+
+        public static string GetDisplayName(MethodInfo method, object attribute)
+        {
+            /*Comparing via full name, because attribute parameter is from "TestFramework.dll"
```

Comment:
minor: better use // rather than /* pattern for the comments. And please add a space after either the // or /*